pkg/cache/v3/cache.go

```diff
@@ -186,7 +186,7 @@ func (r *RawResponse) GetDiscoveryResponse() (*discovery.DiscoveryResponse, erro
 		marshaledResources := make([]*any.Any, len(r.Resources))
 
 		for i, resource := range r.Resources {
-			maybeTtldResource, resourceType, err := r.maybeCreateTtlResource(resource)
+			maybeTtldResource, resourceType, err := r.maybeCreateTTLResource(resource)
```

Comment:
I'd rather keep the TTL stuff consistent. Maybe change `ResourceWithTtl` to `ResourceWithTTL`?

Reply:
This would be `TTL` according to Go's [initialism convention](https://github.com/golang/go/wiki/CodeReviewComments#initialisms). I updated the internal names, but I didn't want to change the API in this PR.

Reply:
Yup I agree. I honestly am okay with changing the API. If that breaks users they can pin, and we also haven't hit `v1.0` yet.

Reply:
Ok, changing it. AFAIK we can't add a compatibility alias to a field so there's no way to rename without a breaking change.